### PR TITLE
[FileProvider] Update bindings to Xcode 11.4 Beta 3

### PR DESCRIPTION
**Pull request by dalexsoto**
We got the deprecated attributes! Long live our test suites!

### CODE REVIEW

File: src/fileprovider.cs [hunk 1]
```diff
@@ -488,6 +488,9 @@ interface NSFileProviderItem {
 		[Export ("favoriteRank", ArgumentSemantic.Copy)]
 		NSNumber GetFavoriteRank ();
 
+#if XAMCORE_4_0 // Not available in mac
+		[NoMac]
+#endif
```
Comment (spouliot) [inline src/fileprovider.cs]: let's add an `[Obsolete]` message (on macOS) so people won't think it should work
*and* won't be surprised if the member is removed later (at `XAMCORE_4_0` time)

Comment (dalexsoto) [reply src/fileprovider.cs]: Good call 👍

Comment (dalexsoto) [inline src/fileprovider.cs]:
```suggestion
#if XAMCORE_4_0 // Not available in mac
		[NoMac]
#elif MONOMAC
		[Obsolete ("'IsTrashed' is not available in macOS and will be removed in the future.")]
#endif
```

Comment (dalexsoto) [reply src/fileprovider.cs]: Fixed, thanks @spouliot

**Pull request merged**